spec/requests/partners_requests_spec.rb

```diff
@@ -15,9 +15,21 @@
   end
 
   describe "GET #show" do
+    let(:partner) { create(:partner, organization: @organization) }
+    let(:fake_get_return) do
+      {
+        family_count: Faker::Number.number
+      }
```

Comment:
```suggestion
      }.to_json
```
Let's mimic a little closer to what is happening in the code. And then in line 32:
```
expect(assigns[:impact_metrics]).to eq(JSON.parse(fake_get_return))
```

This is to add spec coverage for the JSON.parse addition

Reply:
Pushed this change (+ a fix)